Lib/glyphsLib/builder/sources.py

```diff
@@ -80,6 +83,59 @@ def _to_designspace_source(self, master, is_regular):
     source.location = location
 
 
+def _to_designspace_source_layer(self):
+    brace_layers = collections.defaultdict(list)
+    layer_to_glyph = collections.defaultdict(list)
+    for glyph in self.font.glyphs:
+        for layer in glyph.layers:
+            if (
+                "{" in layer.name
+                and "}" in layer.name
+                and ".background" not in layer.name
```

Comment:
what's this ".background" magic string?

Reply:
As you found, brace layers may get background layers that I have to not write out to a Designspace file.

https://github.com/googlei18n/glyphsLib/blob/fba6344b4830ee527c6a52ef2e29330e5cc2abdb/Lib/glyphsLib/builder/layers.py#L44
https://github.com/googlei18n/glyphsLib/blob/fba6344b4830ee527c6a52ef2e29330e5cc2abdb/Lib/glyphsLib/builder/layers.py#L69-L72

Reply:
I just wonder if there's a better way to detect whether a layer is a background layer instead of checking its name has a ".background" suffix. Isn't the latter just the way glyphsLib translates Glyphs background layers to UFO layers? or is this a convention coming from Glyphs.app itself? /cc @belluzj